misc/packaging/README.md

```diff
@@ -0,0 +1,117 @@
+# ubuntuSoS
+this scripts show how to build acrn-hypervisor and acrn-kernel deb package
+
+1. setup native ubuntu
+
+install python3
+
+config proxy
+
+sudo su
+
+
+2. config json file
```

Comment:
Configure which json file?  Are we adding the information below to some existing json file?

Reply:
add json file name and explanation